assets/style.css

```diff
@@ -0,0 +1,600 @@
+  font-weight: 600;
+  margin: 1.5rem 0 1.5rem 0;
+}
+
+/* Main-Program */
+.main-program {
+  background: url("../assets/images/program.svg"), #272a32;
+  width: 100%;
+  text-align: center;
+  padding: 3rem 0;
+  height: 70vh;
+}
+
+.program-labels {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 3rem;
+  gap: 1rem;
+}
+
+.program-items {
+  background: #404144b4;
+  width: 150px;
+  height: 200px;
+}
+
+.program-heading {
+  text-align: center;
+  color: var(--white);
+}
+
+.fa-artstation,
+.fa-wpforms,
+.fa-house-laptop,
+.fa-chalkboard-user {
+  font-size: 2.5rem;
+  color: var(--white);
+  margin-top: 2rem;
+}
+
+.tag {
+  color: #ec240d;
+  font-size: 1rem;
+  font-family: var(--font-headings);
+}
+
+.description {
+  color: var(--white);
+}
+
+.program-heading2 {
+  color: var(--white);
+  text-decoration: underline;
+  text-decoration-color: var(--white);
+}
+
+.program-btn {
+  display: none;
+}
+
+/* Mobile-Version */
+@media screen and (max-width: 768px) {
+  .nav-section {
+    display: none;
+  }
+
+  .hero-section {
+    max-width: 110%;
+    height: 600px;
+    margin: 0 auto;
+    justify-content: center;
+    padding: 20px;
+  }
+
+  .page-heading {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: flex-start;
+    background: transparent;
+    position: absolute;
+    top: 0;
+    margin-top: 0;
+    box-shadow: none;
+  }
+
+  .logo-img {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    top: 0;
+    background: #ec5342ec;
+    height: 110vh;
+    width: 200px;
+    left: -200px;
+    text-align: left;
+    z-index: 2;
+    transition: 1s;
+    -webkit-backdrop-filter: blur(5px);
+    backdrop-filter: blur(5px);
+  }
+
+  .nav-items {
+    flex-direction: column;
+    padding-top: 5rem;
+    line-height: 5rem;
+    letter-spacing: 1.5px;
+  }
+
+  .nav-items li a {
+    font-size: 2rem;
+    color: rgb(225, 225, 225);
+  }
+
+  .nav-items li a:hover {
+    color: #ebc934;
+  }
+
+  .btn {
+    display: flex;
+    justify-content: center;
+    color: rgb(225, 225, 225);
+    border: 2px solid rgb(225, 225, 225);
+    padding: 1rem 2rem;
+    font-size: 1.4rem;
+    margin-top: 1.5rem;
+  }
+
+  i#openBtn,
+  i#closeBtn {
+    display: block;
+  }
+
+  .fa-times {
+    display: block;
+    color: var(--color-primary);
+    margin: 1.5rem;
+    position: absolute;
+    left: -1rem;
+    padding-left: 0.75rem;
+    font-size: 2.2rem;
+    cursor: pointer;
+  }
+
+  .fa-bars {
+    display: block;
+    color: var(--primary-color);
+    font-size: 2.8rem;
+    position: absolute;
+    left: 1rem;
+    cursor: pointer;
+  }
+
+  /* Headline-section */
+  .main-section {
+    margin: 0.1rem 0;
+    height: 80vh;
+    background-image: url(../images/mobile.png);
+  }
+
+  .text {
+    margin-bottom: 2rem;
+  }
+
+  /* Main -program */
+  .main-program {
+    height: 140vh;
+  }
```

Comment:
- [ ] I have noticed that your `main-program section` is also overlaping the `speakers section` for Mobile version. Consider increasing the height for the `.main-program` class. This will help Fix the overlapping issue
 
![image](https://user-images.githubusercontent.com/52143815/175437671-bf24e4eb-73eb-4f91-a94e-2ba59e6887b6.png)